subscriptions/views.py

```diff
@@ -1,26 +1,40 @@
-from rest_framework import viewsets, status
+from rest_framework import filters, viewsets, status
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework.authtoken.models import Token
 from django.core.exceptions import ObjectDoesNotExist
 from django.contrib.auth.models import User
+import django_filters
 
 from .models import Subscription
 from .serializers import SubscriptionSerializer, CreateUserSerializer
 from .tasks import send_next_message, scheduled_metrics
 from seed_stage_based_messaging.utils import get_available_metrics
 
 
+class SubscriptionFilter(filters.FilterSet):
+    created_from = django_filters.IsoDateTimeFilter(
+        name="created_at", lookup_type="gte")
+    created_to = django_filters.IsoDateTimeFilter(
+        name="created_at", lookup_type="lte")
+
+    class Meta:
+        model = Subscription
+        filters = (
+            'identity', 'messageset_id', 'lang', 'active', 'completed',
```

Comment:
Ah, that's a typo that has unintended consequences that cause the tests to pass. It seems like if `fields` and `exclude` aren't present, then it allows you to filter on all fields, which is pretty much what we're specifying here. https://github.com/carltongibson/django-filter/blob/develop/django_filters/filterset.py#L252-L253 .

So I'm going to remove this to allow us to filter on all fields.